pkgs/development/python-modules/ramp-workflow/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, pandas
+, scikitlearn
+, cloudpickle
+, click
+, pytest
+, pytestcov
+, codecov
+, flake8
+, nbconvert
+, jupyter_client
+, seaborn
+, netcdf4
+, xarray
+, scikitimage
+, joblib
+, Keras
+, tensorflow
+}:
+
+buildPythonPackage rec {
+  pname = "ramp-workflow";
+  version = "v0.2.0+53.gf48b024";
```

Comment:
Why not use https://github.com/paris-saclay-cds/ramp-workflow/releases/tag/0.3.1 ?

Reply:
Yes, I should update.